policies/definitions.md

```diff
@@ -0,0 +1,53 @@
+# OpenSSL terms and definitions
+
+## ABI
+
+Application binary interface.
+
+An ABI compatible release allows applications to
+be relinked against the new library with the expectation that everything will
+continue to function.
+
+## API
+
+Application programming interface.
+
+An API compatible release allows application to be recompiled against the new
+library with the expectation that everything will continue to function.
+
+## LTS
+
+Long term stable.  These releases are supported for longer than normal releases.
```

Comment:
The acronym LTS has always meant Long Term Support (i.e. not Stable). See:

https://www.openssl.org/policies/releasestrat.html

Reply:
Done